app/core/aws.py

```diff
@@ -45,17 +45,45 @@ def from_url(cls, url: str) -> "S3Document":
 class S3Client:
     """Helper class to connect to S3 and perform actions on buckets and documents."""
 
-    def __init__(self):  # noqa: D107
-        self.client = boto3.client(
-            "s3",
-            aws_access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
-            aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
-            config=botocore.client.Config(
-                signature_version="s3v4",
-                region_name=AWS_REGION,
-                connect_timeout=10,
-            ),
-        )
+    def __init__(self, dev_mode: bool):  # noqa: D107
+        if dev_mode is True:
+            logger.debug("***************** IN DEVELOPMENT MODE *****************")
```

Comment:
I'm wondering if this should always come out - regardless of which level of logging we have enabled - so that even in production with loglevel of info this would appear?